app/views/dashboard/_header.html.erb

```diff
@@ -1,14 +1,15 @@
   <div class="row header">
 
     <% if current_user %>
-      <h1><%= t('dashboard._header.dashboard') %></h1>
+      <h4><%= t('dashboard._header.dashboard') %></h4>&nbsp&nbsp
+      <p style="color:#808080; margin-top:2px;">Updates from community members on <a href="/subscriptions" style="color:#808080;"><u> topics you follow </u></a></p>
     <% else %>
-      <h1><%= t('dashboard._header.community_research') %></h1>
+      <h4><%= t('dashboard._header.community_research') %></h4>
     <% end %>
 
   </div>
 
-<div class="row stats">
+<!-- <div class="row stats">
   <h4 class="col-lg-8">
```

Comment:
Hi @gautamig54 May I know why is this section commented ? :thinking:

Reply:
We do not have the welcome text as such in the new design. It is more like an alert message. I'll add the alert message text. Should I completely delete this part?